kingfisher_scrapy/spiders/kyrgyzstan.py

```diff
@@ -18,11 +21,18 @@ class Kyrgyzstan(LinksSpider):
     """
     name = 'kyrgyzstan'
 
+    # BaseSpider
+    date_format = 'datetime'
+
     # SimpleSpider
     data_type = 'release_package'
 
     # LinksSpider
     formatter = staticmethod(parameters('offset'))
 
     def start_requests(self):
-        yield scrapy.Request('http://ocds.zakupki.gov.kg/api/tendering', meta={'file_name': 'offset-0.json'})
+        url = 'http://ocds.zakupki.gov.kg/api/tendering'
+        if self.from_date:
+            # The API requires the timezone and seconds in the since parameter
+            url = f'{url}?since={self.from_date.strftime(self.date_format)}.00%2B06:00'
+        yield scrapy.Request(url, meta={'file_name': 'offset-0.json'})
```

Comment:
We aren't using an offset param.

```suggestion
        yield scrapy.Request(url, meta={'file_name': 'start.json'})
```

Reply:
offset=0 is the default but OK

Reply:
Where is it the default?